tests/testthat.R

```diff
@@ -0,0 +1,9 @@
+library(testthat)
+library(PythonEmbedInR)
+
+test_check("PythonEmbedInR")
+
+# uninstall pandas
+print("Uninstalling pandas ...")
+package_dir <- gsub("(PythonEmbedInR).*", "\\1", getwd())
```

Comment:
for those of us weak in regexp's, please add a comment explaining what this does, as well as a couple tests that exercise it.